rte/main.go

```diff
@@ -71,6 +73,15 @@ func main() {
 		os.Exit(0)
 	}
 
+	if parsedArgs.DumpConfig {
+		data, err := json.Marshal(parsedArgs)
+		if err != nil {
+			klog.Fatalf("encoding the configuration to JSON")
+		}
+		fmt.Printf("%s\n", string(data))
```

Comment:
can we add the values of the environment variables as well?

Reply:
uhm, what's the use case for that?